

arch/ia64/kernel/domain.c: In function `sched_domain_node_span':
arch/ia64/kernel/domain.c:80: warning: unused variable `next_node'
arch/ia64/kernel/domain.c: In function `arch_init_sched_domains':
arch/ia64/kernel/domain.c:143: warning: unused variable `node'
arch/ia64/kernel/domain.c: At top level:
arch/ia64/kernel/domain.c:67: warning: `sched_domain_node_span' defined but not used

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/ia64/kernel/domain.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN arch/ia64/kernel/domain.c~ia64-sched_domains-warning-fixes arch/ia64/kernel/domain.c
--- 25/arch/ia64/kernel/domain.c~ia64-sched_domains-warning-fixes	Mon Oct  4 13:41:48 2004
+++ 25-akpm/arch/ia64/kernel/domain.c	Mon Oct  4 13:48:12 2004
@@ -15,6 +15,7 @@
 
 #define SD_NODES_PER_DOMAIN 4
 
+#ifdef CONFIG_NUMA
 /**
  * find_next_best_node - find the next node to include in a sched_domain
  * @node: node whose sched_domain we're building
@@ -80,7 +81,7 @@ static cpumask_t __devinit sched_domain_
 
 	return span;
 }
-
+#endif
 
 /*
  * At the moment, CONFIG_SCHED_SMT is never defined, but leave it in so we
_
